pages/token/styles.module.scss

```diff
@@ -115,3 +115,7 @@
     height: 1.25rem;
   }
 }
+
+.item-dd-cls {
```

Comment:
Is it necessary to add a class instead of using nested CSS selectors? If sure, please use a meaningful class name

Reply:
@JeffreyMa597 please have a review on this question

Reply:
@Keith-CY changed.